ivy/functional/frontends/numpy/logic/truth_value_testing.py

```diff
@@ -40,3 +40,11 @@ def isscalar(element):
         or isinstance(element, bool)
         or isinstance(element, float)
     )
+
+@to_ivy_arrays_and_back   
+def iscomplex(element):
+    ax = ivy.asarray(element)
+    if issubclass(ax.dtype.type, ivy.complexfloating):
+        return ax.imag != 0
+    res = ivy.zeros(ax.shape, bool)
+    return res[()]
```

Comment:
There are some lint errors. Did you install the pre-commit hook? Also, I'll appreciate it if you tag me once you finish editing your PR; so I know it's ready. Thanks! 😊

Reply:
Hi @ZoeCD ,
I have installed the pre-commit hook but I don't know if it's a suitable version.Can please check  ,once again?

Reply:
Just don't forget to leave a line at the end of the file

Reply:
Ok.🙂.Thank you for reviews.